closed/custom/common/JdkNativeCompilation-pre.gmk

```diff
@@ -1,5 +1,5 @@
 # ===========================================================================
-# (c) Copyright IBM Corp. 2018, 2020 All Rights Reserved
+# (c) Copyright IBM Corp. 2018, 2025 All Rights Reserved
```

Comment:
Is this copyright year change required?

Reply:
This file was renamed: I thought it safer to consider that a change.